src/libs/actions/OnyxDerived/index.ts

```diff
@@ -39,6 +39,7 @@ function init() {
                         sourceValues: undefined,
                         areAllConnectionsSet: false,
                     };
+                    // @ts-expect-error TypeScript can't confirm the shape of dependencyValues matches the compute function's parameters
```

Comment:
hmmm this seems suspicious. Why has this not been a problem until now?

Reply:
Actually it already has been an issue before, those comments were recently removed in this PR: https://github.com/Expensify/App/pull/62653. I spent some time digging into it in the past, but I couldn't figure it out